submissions/benjaminKamena/phpDataase/ordersQueryDb.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+include('connected.php');
+$id =  $_GET['id'];
+
+function view($id){
+	try{
+		$sql = "SELECT order_items.quantity, order_items.unit_price, 
+            order_items.total_amount, orders.order_date, orders.id,customers.street_address, customers.city,customers.state, 
+            customers.country,products.name 
+            FROM order_items 
+            LEFT JOIN orders ON orders.id = order_items.order_id
+            LEFT JOIN customers on orders.customer_id = customers.id
+            LEFT JOIN products ON products.id = order_items.product_id 
+            WHERE orders.customer_id = '$id' ";
+            $stmt = $pdo->query($sql);
+            $orders = $stmt->fetchAll();
+            setcookie("orders",  serialize($orders), time() + 3600 );
+        }catch (PDOException $e) {
+            die("ERROR: Could not execute $sql. " . $e->getMessage());
+        }
+    };
+
+    if(isset($_COOKIE['orders'])){
+        $orders =  unserialize($_COOKIE['orders']);
+    }
```

Comment:
We are making progress. But, If the cookie is not set, what should happen?

Reply:
Am using cookies from previous task

Reply:
I don't get you. But, my question is if the cookie `orders` is not  where will the `$orders` be coming from?